docs/configuration.md

```diff
@@ -2306,6 +2306,50 @@ showDF(properties, numRows = 200, truncate = FALSE)
   </tr>
 </table>
 
+### Spark Application Garbage Collector
+
+<table class="table">
+  <tr><th>Property Name</th><th>Default</th><th>Meaning</th></tr>
+  <tr>
+    <td><code>spark.cleaner.periodicGC.interval</code></td>
+    <td>30min</td>
+    <td>
+      Controls how often to trigger a garbage collection.
```

Comment:
You might give at least a sentence of explanation about what this has to do with the context cleaner.

Reply:
Given more detailed description.